src/data/moves/move.ts

```diff
@@ -6823,6 +6823,7 @@ export class RandomMovesetMoveAttr extends CallMoveAttr {
   }
 }
 
+// TODO: Shouldn't this be `CallMoveAttr`?
```

Comment:
The answer is yes lol.
```suggestion
// TODO: This should extend `CallMoveAttr`
```